data/data/powervs/providers.tf

```diff
@@ -4,7 +4,15 @@ terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = "~> 1.24.0"
+      version = "1.25.0"
+    }
+    ignition = {
+      source = "community-terraform-providers/ignition"
+      version = "2.1.2"
+    }
+    presign = {
+      source = "ibm/openshift-powervs/presign"
+      version = "0.0.1"
```

Comment:
do we really need this here?

Reply:
This was the only way I was able to get the provider to work. I followed the "Third-party Providers" section of [this page](https://www.hashicorp.com/blog/automatic-installation-of-third-party-providers-with-terraform-0-13)